<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialogAlternative</class>
 <widget class="QDialog" name="QWDialogAlternative">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改翻译</string>
  </property>
  <widget class="QPushButton" name="btnAccept">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReject">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Item">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>替换项目</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_original">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>原文</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_alternative">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Order">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>替换顺序</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Alternative">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_Original">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>还不满意你可以自己在修改框中输入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAccept</sender>
   <signal>clicked()</signal>
   <receiver>QWDialogAlternative</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReject</sender>
   <signal>clicked()</signal>
   <receiver>QWDialogAlternative</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
